docs/csharp/language-reference/operators/remainder-operator.md

```diff
@@ -1,28 +1,32 @@
 ---
 title: "% Operator (C# Reference)"
-ms.date: 07/20/2015
+ms.date: 04/03/2018
 ms.prod: .net
 ms.technology: 
   - "devlang-csharp"
 ms.topic: "article"
 f1_keywords: 
   - "%_CSharpKeyword"
 helpviewer_keywords: 
-  - "modulus operator [C#]"
+  - "remainder operator [C#]"
   - "% operator [C#]"
 ms.assetid: 3b74f4f9-fd9c-45e7-84fa-c8d71a0dfad7
 caps.latest.revision: 15
 author: "BillWagner"
 ms.author: "wiwagn"
 ---
 # % Operator (C# Reference)
-The `%` operator computes the remainder after dividing its first operand by its second. All numeric types have predefined remainder operators.  
+The remainder operator (`%`) computes the remainder after dividing its first operand by its second. All numeric types have predefined remainder operators. 
   
 ## Remarks  
+ The formula `a % b` will always return a value on the range `(-b, b)`, exclusive (it can never return `b` or `-b`), keeping the sign of the dividend. The remainder operator satisfies the rule where if `q = a / b` and `r = a % b`, then `a = q * b + r`.
```

Comment:
IMO it would be preferable to use the same definition that is used in both the C# spec and the IL spec - `a % b = a - (a / b) * b`.

Reply:
> BTW, where is that formula in the C# spec?

I'm not sure I understand the question. It seems that you already found it.